src/integrationtests/java/com/aws/greengrass/integrationtests/ipc/IPCServicesTest.java

```diff
@@ -136,6 +143,16 @@ void beforeEach(ExtensionContext context) {
         ignoreExceptionOfType(context, InterruptedException.class);
     }
 
+    @Test
+    void Given_assign_path_for_ipcSocket_When_startUp_Then_ipcSocket_store_in_assigned_path() {
```

Comment:
this test isn't testing using a customized path. 

You must add a test case which covers the case when the user chooses to put the socket file somewhere else.